? .swp
? iconv-detect.h
? calendar/backends/weather
? camel/camel-mime-tables.c
? libedataserverui/test-contact-store
? libedataserverui/test-name-selector
? src/GNOME_Evolution_DataServer_1.2.server
Index: configure.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/configure.in,v
retrieving revision 1.83
diff -u -r1.83 configure.in
--- configure.in	4 Jan 2005 18:47:47 -0000	1.83
+++ configure.in	7 Jan 2005 00:10:00 -0000
@@ -1137,6 +1137,9 @@
 gladedir="$privdatadir/glade"
 AC_SUBST(gladedir)
 
+weatherdatadir="$privdatadir/weather"
+AC_SUBST(weatherdatadir)
+
 dnl separate camel from e-d-s?  or should it be under the same spot?  same for now.
 camel_providerdir="${privlibdir}/camel-providers"
 AC_SUBST(camel_providerdir)
@@ -1204,6 +1207,7 @@
 calendar/backends/groupwise/Makefile
 calendar/backends/http/Makefile
 calendar/backends/contacts/Makefile
+calendar/backends/weather/Makefile
 calendar/tests/Makefile
 calendar/tests/ecal/Makefile
 camel/Makefile
Index: evolution-data-server.pc.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/evolution-data-server.pc.in,v
retrieving revision 1.2
diff -u -r1.2 evolution-data-server.pc.in
--- evolution-data-server.pc.in	30 Sep 2004 16:53:54 -0000	1.2
+++ evolution-data-server.pc.in	7 Jan 2005 00:10:00 -0000
@@ -4,6 +4,8 @@
 includedir=@includedir@
 privlibdir=@privlibdir@
 extensiondir=@extensiondir@
+datadir=@datadir@
+privdatadir=@privdatadir@
 
 idldir=@idldir@
 IDL_INCLUDES=-I ${idldir} @IDL_INCLUDES@
Index: calendar/ChangeLog
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/ChangeLog,v
retrieving revision 1.368
diff -u -r1.368 ChangeLog
--- calendar/ChangeLog	6 Jan 2005 07:33:54 -0000	1.368
+++ calendar/ChangeLog	7 Jan 2005 00:10:00 -0000
@@ -1,3 +1,7 @@
+2005-01-06  David Trowbridge <trowbrds@cs.colorado.edu>
+
+	* backends/weather/*: Import weather calendar backend
+
 2005-01-06 Chenthill Palanisamy <pchenthill@novell.com>
 	
 	* backends/groupwise/e-cal-backends-groupwise-utils.[ch]:
Index: calendar/backends/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/backends/Makefile.am,v
retrieving revision 1.3
diff -u -r1.3 Makefile.am
--- calendar/backends/Makefile.am	9 Jan 2004 20:05:27 -0000	1.3
+++ calendar/backends/Makefile.am	7 Jan 2005 00:10:00 -0000
@@ -1 +1 @@
-SUBDIRS = file groupwise http contacts
+SUBDIRS = file groupwise http contacts weather
